community/eula_schinese.txt

```diff
@@ -1,3 +1,3 @@
 Alien Swarm: Reactive Drop - 最终用户许可协议
```

Comment:
should the title also be changed?

Reply:
This non-game content (organization names, logos, logos, some agreements) is not translated. I think these contents are similar to certain legal benefits, so there is no translation. Of course, if you confirm that you need translation, I can adjust it synchronously.